test/unstable/sortInterfaces.good

```diff
@@ -0,0 +1,4 @@
+sortInterfaces.chpl:1: warning: The Sort module interface is unstable
```

Comment:
I thought we got rid of this line in your other branch?

Reply:
I haven't merged that PR yet

Reply:
Okay, cool.  Just make sure to merge main in and retest whichever one gets merged second!